zanata-model/src/main/java/org/zanata/model/HAccountRole.java

```diff
@@ -74,6 +74,7 @@ public String getName() {
         return groups;
     }
 
+    // TODO [CDI] check whether we actually use this field (doesn't seem to in prod db and in code)
     @RoleConditional
     public boolean isConditional() {
```

Comment:
On 2015-07-30 13:17, Carlos Munoz wrote:
> In zanata-model/src/main/java/org/zanata/model/HAccountRole.java
> <https://github.com/zanata/zanata-server/pull/908#discussion_r35835150>:
> 
>> @@ -74,6 +74,7 @@ public String getName() {
>>          return groups;
>>      }
>>  
>> +    // TODO [CDI] check whether we actually use this field (doesn't seem to in prod db and in code)
>>      @RoleConditional
>>      public boolean isConditional() {
> 
> No idea to be honest. What does the javadoc say?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/908/files#r35835150>.
> 

Naturally, it says "Flags an entity field or method as representing
whether a role is conditional or not."  Isn't that enlightening?

Reading JpaIdentityStore.listGrantableRoles I suspect it could be used
to implement the FAS role, or perhaps project-dependent roles like
project-maintainer or project-translator.

I think we should leave it in for now.

-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat